elasticdl/python/master/master.py

```diff
@@ -76,6 +63,30 @@ def prepare(self):
         self._master_server.start()
         logger.info("Master RPC server started")
 
+    def _set_pod_manager(self):
+        if self.elasticdl_job_service:
+            command = self.elasticdl_job_service.get_ps_worker_command()
+            self.pod_manager.set_up(
+                worker_command=command,
+                worker_args=self.elasticdl_job_service.get_worker_args(
+                    self._args
+                ),
+                ps_command=command,
+                ps_args=self.elasticdl_job_service.get_ps_args(self._args),
+            )
+        elif self._args.job_command:
+            self.pod_manager.set_up(
```

Comment:
Also, `set_up` can rename to `set_pod_command`.

Reply:
You can see the discussion about the function name. https://github.com/sql-machine-learning/elasticdl/pull/2386#discussion_r531547517